services/core-api/app/api/verifiable_credentials/manager.py

```diff
@@ -195,3 +288,116 @@ def produce_map_01_credential_payload(cls, did: str, permit_amendment: PermitAme
+            return None
+
+        untp_party_cpo = base.Party(
+            name="Chief Permitting Officer",
+            identifiers=[
+                base.Identifier(
+                    scheme=ANONCRED_SCHEME,
+                    identifierValue="did:indy:candy:A2UZSmrL9N5FDZGPu68wy",
+                    identifierURI="https://candyscan.idlab.org/tx/CANDY_PROD/domain/321",
+                    verificationEvidence=base.Evidence(
+                        format=codes.EvidenceFormat.W3C_VC,
+                        credentialReference=
+                        "https://candyscan.idlab.org/tx/CANDY_PROD/domain/321"            #this is an anoncred
+                    ))
+            ])
+        orgbook_cred_url = f"https://orgbook.gov.bc.ca/entity/{orgbook_entity.registration_id}/credential/{orgbook_entity.credential_id}"
+
+        untp_party_business = base.Party(
+            name=orgbook_entity.name_text,
+            identifiers=[
+                base.Identifier(
+                    scheme=ANONCRED_SCHEME,
+                    identifierValue=str(orgbook_entity.registration_id),
+                    identifierURI=orgbook_cred_url,
+                    verificationEvidence=base.Evidence(
+                        format=codes.EvidenceFormat.W3C_VC, credentialReference=orgbook_cred_url))
+            ])
+
+        facility = cc.Facility(
+            name=permit_amendment.mine.mine_name,
+            geolocation=
+            f'https://plus.codes/{plus_code_encode(permit_amendment.mine.latitude, permit_amendment.mine.longitude)}',
+            verifiedByCAB=True)
+
+        products = [
+            cc.Product(
+                name=c,
+                classifications=cc.Classification(
+                    scheme="https://unstats.un.org/unsd/classifications/Econ/cpc",
+                    classifierName=c),
+                verifiedByCAB=False) for c in permit_amendment.mine.commodities
+        ]
+
+        untp_assessments = [
+            cc.ConformityAssessment(
+                referenceRegulation=cc.Regulation(
+                    id="https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01",
+                    name="BC Mines Act",
+                    issuingBody=base.Party(
+                        name="BC Government",
+                        identifiers=[
+                            base.Identifier(identifierValue="did:indy:candy:LTNyw5R14J66CrF7tmV3i8")
+                        ]),
+                    effectiveDate=datetime(2024, 5, 14, tzinfo=pytz.timezone("UTC")).isoformat()),
+                subjectFacilities=[facility],
+                subjectProducts=products,
+                sustainabilityTopic=codes.SustainabilityTopic.Governance_Compliance)
+        ]
+        issue_date = permit_amendment.issue_date
+        issuance_date_str = datetime(
+            issue_date.year, issue_date.month, issue_date.day, 0, 0, 0,
+            tzinfo=pytz.timezone("UTC")).isoformat()
```

Comment:
According to Sonarcloud, it's preferred to use `datetime.tzinfo` instead of `pytz.timezone` when using `datetime` as there's a slight difference in how timezones are handled between them, hard to say if it'll actually have an impact here. https://sonarcloud.io/project/issues?resolved=false&pullRequest=3162&id=bcgov-sonarcloud_mds_core-api&open=AZAyy_vxY_vcUsn0Ofd6&tab=why

Reply:
that warning recommends using zoneinfo package if python version 3.9 or greater. we are using python 3.8 right?

Reply:
We're actually on python 3.11 😄

Reply:
ok, well i rebuild my venv and am using zoneinfo now.